src/spec.rs

```diff
@@ -67,7 +68,8 @@ impl<'a> Spec<'a> {
         let (md_document, lines) = metadata::parse_metadata(&self.lines);
         self.lines = lines;
 
-        let mut md = Metadata::new();
+        // TODO: avoid cloning metadata here
+        let mut md = self.md_cli.clone();
```

Comment:
Do we really want the CLI metadata have less priority than the document one?

Reply:
Oops, I will fix that in the next PR.